static/src/javascripts/projects/commercial/modules/prebid/prebid.js

```diff
@@ -196,8 +196,14 @@ class PrebidService {
                                 const bidResponses = window.pbjs.getBidResponses()[
                                     advert.id
                                 ];
+                                const adSlotBidResponses =
+                                    bidResponses && advert.id in bidResponses
```

Comment:
Could potentially use hasOwnProperty instead? 🤔 

`bidResponses  && bidResponses.hasOwnProperty(advert.id)`

Reply:
Yes, good shout! Not sure what's better here. Here we just care that it exists on the object or not! Using `hasOwnProperty` is assuming the object definitely does not inherit the key and we can't say for sure if that's true or not. (Although I imagine it's a fairly safe assumption to make! :smile: )